conformance/utils/kubernetes/apply.go

```diff
@@ -266,3 +340,20 @@ func getContentsFromPathOrURL(fs embed.FS, location string, timeoutConfig config
 	}
 	return bytes.NewBuffer(b), nil
 }
+
+// convertGatewayAddrsToPrimitives converts a slice of Gateway addresses and
+// to a slice of primite types and then returns them as a []interface{} so that
```

Comment:
```suggestion
// convertGatewayAddrsToPrimitives converts a slice of Gateway addresses
// to a slice of primitive types and then returns them as a []interface{} so that
```